botocore/httpsession.py

```diff
@@ -224,6 +247,26 @@ def _setup_ssl_cert(self, conn, url, verify):
             conn.cert_reqs = 'CERT_NONE'
             conn.ca_certs = None
 
+    def _setup_proxy_ssl_context(self, proxies_settings):
+        proxy_ca_bundle = proxies_settings.get('proxy_ca_bundle')
+        proxy_cert = proxies_settings.get('proxy_client_cert')
+        if proxy_ca_bundle is None and proxy_cert is None:
+            return None
+
+        context = self._get_ssl_context()
+        try:
+            if proxy_ca_bundle is not None:
+                context.load_verify_locations(capath=proxy_ca_bundle)
```

Comment:
Did you mean ``cafile`` instead of ``capath`` given that we're talking about a bundle? (generally this a single file containing root/intermediary certificates)

Reply:
Great catch, @jalopezsilva. Latest revision should have that updated and I think we're pretty close to getting this merged.